portal-api/src/v1/routes.js

```diff
@@ -28,25 +27,50 @@ const { ukefDecisionReport, unissuedFacilitiesReport } = require('./controllers/
 const dealImportBssEwcsController = require('./controllers/data-migration/deal-import-bss-ewcs.controller');
 const dealImportGefController = require('./controllers/data-migration/deal-import-gef.controller');
 
-const users = require('./users/routes');
-const { cleanXss, fileUpload } = require('./middleware');
+const { fileUpload } = require('./middleware');
 
+const users = require('./users/routes');
 const gef = require('./gef/routes');
 
-const authRouter = express.Router();
 const openRouter = express.Router();
+
+openRouter.route('/login').post(users.login);
+
+// 1. Request password reset
+openRouter.route('/users/reset-password').post(users.resetPassword);
+
+// 2. Password reset post request
+openRouter.route('/users/reset-password/:resetPwdToken').post(users.resetPasswordWithToken);
+
 const authRouterAllowXss = express.Router();
 
-authRouterAllowXss.use(passport.authenticate('jwt', { session: false }));
+authRouterAllowXss.route('/feedback').post(feedback.create);
```

Comment:
This request and the ones on lines 49-51 previously used `openRouter` so didn't have the JWT check

Reply:
I've got a similar one on TFM - this means the UI page also needs to be not accessible when not logged in and redirect when logged in. Haven't fixed this on my branch yet but just a heads up :)

Reply:
Resolving this thread as we're now keeping the page visible to non-logged in users and using an api key as authentication